src/OrbitGl/GpuTrack.cpp

```diff
@@ -114,16 +114,16 @@ float GpuTrack::GetHeight() const {
   }
   return height;
 }
+// TODO(): Make a general interface for tracks for setting the width for every child.
+void GpuTrack::SetWidth(float width) {
+  Track::SetWidth(width);
+  submission_track_->SetWidth(GetWidth());
+  marker_track_->SetWidth(GetWidth());
```

Comment:
I assume `GetWidth()` is equal to `width`?

Nit: Maybe also use `width` for `submission_track` and `marker_track`. This would also make it consistent to `ThreadTrack::SetWidth` from below.

Reply:
Yes, of course. Thanks for pointing it out!